developer-workflow/grammar.rst

```diff
@@ -31,15 +31,16 @@ Note: sometimes things mysteriously don't work.  Before giving up, try ``make cl
   (This runs Python's parser generator, ``Tools/peg_generator``).
 
 * :cpy-file:`Grammar/Tokens` is a place for adding new token types.  After
-  changing it, run ``make regen-token`` to regenerate :cpy-file:`Include/token.h`,
-  :cpy-file:`Parser/token.c`, :cpy-file:`Lib/token.py` and
-  :cpy-file:`Doc/library/token-list.inc`.  If you change both ``python.gram``
-  and ``Tokens``, run ``make regen-token`` before ``make regen-pegen``.
+  changing it, run ``make regen-token`` to regenerate
+  :cpy-file:`Include/internal/pycore_token.h`, :cpy-file:`Parser/token.c`,
```

Comment:
`Include/token.h` was renamed to `Include/internal/pycore_token.h` (the rest is paragraph reflow).